azure/eventhub_log_forwarder/event_hub.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "namespaceName": {
+      "type": "string",
+      "defaultValue": "datadog-eventhub-namespace",
+      "metadata": {
+        "description": "Name of EventHub namespace"
+      }
+    },
+    "eventhubSku": {
+      "type": "string",
+      "allowedValues": [ "Basic", "Standard" ],
+      "defaultValue": "Standard",
+      "metadata": {
+        "description": "Specifies the messaging tier for the EventHub Namespace."
+      }
+    },
+    "skuCapacity": {
+      "type": "int",
+      "allowedValues": [ 1, 2, 4 ],
+      "defaultValue": 1,
+      "metadata": {
+        "description": "MessagingUnits for premium namespace"
+      }
+    },
```

Comment:
I think maybe to make this a bit more simple we can just use some kind of default for these instead of adding more parameters